### ISSUE TITLE
No action hook available to add the content after cart item meta in order table on checkout page

### ISSUE THREAD

**Issue by kartikparmar** [woothemes/woocommerce#36379]
Currently, it is not possible to add the HTML content after the cart item meta in the Order table on the checkout page:

![image](https://user-images.githubusercontent.com/937723/231795718-db7969a3-6f97-4eeb-b0d8-816706de6592.png)

In the above screenshot, I want to add some custom text after the item meta. When I look at the below file, there is no hook available to achieve this.

File: `../plugins/woocommerce/templates/checkout/review-order.php`

![image](https://user-images.githubusercontent.com/937723/231795896-9aeb6315-7b0a-4718-b4b4-1b0d67028504.png)

It would be great if the action hook as below will be added at that place that allows adding any custom elements in the flow.
```
do_action( 'woocommerce_review_order_after_cart_contents', $cart_item['data'], $cart_item, $cart_item_key );
```

**Comment by jorgeatorres** [comment]
Hi @kartikparmar!

Thanks for the report. We agree that adding some flexibility to this file (other than having to override the whole template) makes sense.
We've marked it as an enhancement but it might be a while until we get to this. We will be more than happy to review a PR implementing the changes, if you're up for it.

Thanks again!

**Comment by chetanupare** [comment]
Hello! Is this still open? I’d love to work on it.

**Comment by jorgeatorres** [comment]
@chetanupare: Yes, it is still open. Feel free to contribute a PR and will get it reviewed!

**Comment by chetanupare** [comment]
@jorgeatorres  ,
Hello! I’m new to contributing and have opened a PR for this issue. Any feedback would be greatly appreciated. Thanks!

**Comment by mikejolley** [comment]
@senadir What should we do with requests like these? Obviously adding a hook is trivial but it widens the gap between legacy and block based checkout where parity will drift.

**Comment by pierorocca** [comment]
> [@senadir](https://github.com/senadir) What should we do with requests like these? Obviously adding a hook is trivial but it widens the gap between legacy and block based checkout where parity will drift.

@mikejolley @senadir this is fine for classic checkout. Once we're able to get to the checkout block and extensibility overhaul, we'll have plenty of extensibility points where parity drift won't be a concern. Let me know if I'm misreading the scenario.
